js/hitbtc3.js

```diff
@@ -474,6 +474,8 @@ module.exports = class hitbtc3 extends Exchange {
                     'network': network,
                     'fee': fee,
                     'active': active,
+                    'deposit': payinEnabled,
+                    'withdraw': payoutEnabled,
```

Comment:
We should use `payinEnabledNetwork` and `payoutEnabledNetwork` here, cause this is a dictionary for a network inside a currency, not a dictionary for the currency itself.

Also we should add `deposit` and `withdraw` to the currency structure below.